Aztec/Classes/Extensions/UIColor+Parsers.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+extension UIColor {
+
+    /// Creates a color based on a hexString. If the string is not a valid hexColor it return nils
+    /// Example of colors: #FF0000, #00FF0000
+    ///
+    convenience init?(hexString: String) {
```

Comment:
Can you add a few unit tests for this method?  At least for RGB 24 bit, which I believe is the most common one.